recipes/gtest/all/patches/gtest-cci.20210126-internal-utils.patch

```diff
@@ -0,0 +1,35 @@
+--- a/googletest/cmake/internal_utils.cmake	2021-05-14 08:37:58.099000000 +0200
++++ b/googletest/cmake/internal_utils.cmake	2021-05-14 08:38:50.449675298 +0200
+@@ -280,26 +280,26 @@
```

Comment:
If adding support for this was less intrusive and less labor intensive, I would have been much in favor of allowing this PR to be merged.

However, the "community driven CMake" behind GTest CMake changes extremely frequently... [a new version would need a new patch](https://github.com/google/googletest/blob/9741c42200b66abc708ee6da269a29c8bd912cee/googletest/cmake/internal_utils.cmake#L280) and since it's `off` by default it's very likely (because we have done this mistake in the past) that a new patch would not be generated causing these options to be broken.

Unfortunately, this relies on human intervention which has been very error prone